src/js/pages/WebController/index.js

```diff
@@ -274,26 +273,18 @@ export default class WebControllerPage extends React.PureComponent {
                   pattern="[A-Z,a-z]{3}-[A-Z,a-z]{3}"
                   onKeyUp={e => {
                     const typedValue = e.currentTarget.value;
-                    if (isMobile) {
```

Comment:
@yadvirkaur I can see there is difference in the logic in Mobile and Desktop. Is it something reconsidered now?

Reply:
**Previously:**

**For the Bani Controller:**

- **Mobile Devices:** The code checked if the user had typed four characters without a hyphen and then inserted a hyphen after the third character.

- **Other Devices:** The code added a hyphen automatically after the third character.

**For Sangat Sync:**

The code for all devices added a hyphen automatically after the third character.

**Now:**
I've now unified the logic for the Bani Controller to ensure consistent behaviour across all devices. The code now adds a hyphen automatically after the third character for all devices and ensures that no extra hyphens are added.

Reply:
@yadvirkaur cool, thanks for the explainations, I totally got it. It's just I am unaware of these changes, if these are the changes requested lets do it, and also I would say lets please make sure these are the part mentioned in the ticket, easier to review and followup. 
Otherwise I would request to kindly confirm, as I am sure there might be good reason for this seperation.

Reply:
Totally! 
The mobile-specific logic for the Bani controller is fine as it is since it adds a hyphen after 4 characters only if the user hasn't already added one, so no risk of double hyphens there.

However, this can cause confusion between the Bani Controller and Sangat Sync since they handle hyphens differently for mobile devices.

@saintsoldierx @Gauravjeetsingh veerji please confirm if we need separation of logic or if we can have a unified logic.

Reply:
I don't remember the reason for separate logic, but if its working with the unified logic, let's keep that. Realistically it makes sense to have same logic for both mobile and desktop browsers.